src/styles/stars.scss

```diff
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    width: $starSize;
+    height: $starSize;
+    margin-right: 4px;
+
+    &:last-child {
+      margin-right: 0;
+    }
+
+    @for $starsCount from 1 through 5 {
+      & .star--#{$starsCount}:nth-child(-n + #{$starsCount}) {
+        background-image: url(../images/star-active.svg);
+      }
+    }
+  }
+}
```

Comment:
for some reason your modifier hasn't applied. Please fix it.

Reply:
When you say modifier, are you referring to the active-star for loop? Because if so, I'll be honest, I've been banging my head against the wall with this one. I simply can't figure out how to get the active-star image to apply to the card. Even when I have the code written out in basic form and not a 'for' loop, it still doesn't work...

Reply:
@anastasiia-tilikina 
I think I've fixed it! Not to worry!